src/popup/locales/en.json

```diff
@@ -315,6 +315,7 @@
     },
     "receive": {
       "title": "Receive funds to public address",
+      "titleBtc": "Receive Bitcoin to public address",
```

Comment:
I strongly suggest to avoid using separate translations for each protocols. Imagine we have 20 protocols and 50 translations spread across translations file. I suggest to use translate params.